recipes/shapeshifter/meta.yaml

```diff
@@ -1,28 +1,42 @@
 {% set name = "shapeshifter" %}
-{% set version = "0.0.3" %}
+{% set version = "1.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
```

Comment:
You are overdoing it here. The Jinja2 variables don't actually help with machine processing the recipe. I'd suggest dropping all but the version number. The name won't change, nor will file_ext or hash_type. The hash itself only occurs in one location in the recipe, so having a variable won't improve readability.

Reply:
These were automatically generated using the "conda skeleton pypi" command as suggested in the "Guidelines for bioconda recipes" page. This is how the file was created.